tokens/src/core/components/Carousel.json

```diff
@@ -29,38 +29,33 @@
     "carousel": {
       "base": {
         "source": "$carousel-transition",
-        "$value": [
-          {
-            "property": "opacity",
-            "duration": "0ms",
-            "delay": "0ms",
-            "timingFunction": "ease"
-          },
-          {
-            "property": "transform",
-            "duration": "{transition.carousel.duration}",
-            "delay": "0ms",
-            "timingFunction": "ease-in-out"
-          }
-        ]
+        "$value": {
+          "property": "transform",
+          "duration": "{transition.carousel.duration}",
+          "timingFunction": "ease-in-out",
+          "delay": "0ms",
+          "behavior": "normal"
+        }
       },
       "duration": { "source": "$carousel-transition-duration", "$value": ".6s", "$type": "duration" },
       "indicator": {
         "source": "$carousel-indicator-transition",
         "$value": {
           "property": "opacity",
           "duration": "{transition.carousel.duration}",
+          "timingFunction": "ease",
           "delay": "0ms",
-          "timingFunction": "ease"
+          "behavior": "normal"
```

Comment:
[inform]: Some transitions tokens were missing the `behavior` parameter, now all tokens that are used to add a transitive have the same parameters:
- property
- duration
- timingFunction
- delay
- behavior